README.md

```diff
@@ -4,14 +4,13 @@ MUSIC BLOCKS
 Music Blocks is a collection of manipulative tools for exploring
 fundamental musical concepts in an integrative and fun way.
 
-<img
```

Comment:
Wow. I had no idea this was happening on Windows. I suppose we should fix it throughout the site.